scoresheets/ServeBreakfast.tex

```diff
@@ -2,24 +2,27 @@
 
 \begin{scorelist}
 	\scoreheading{Main Goal}
-	\scoreitem[4]{15}{Picking up breakfast items for transportation to the table}
-	\scoreitem[4]{60}{Placing breakfast items on the table}
-	\scoreitem{300}{Pouring cereal into the bowl}
+	\scoreitem{15}{Initial navigation to pick up area}
+	\scoreitem[4]{15}{Perceiving object and categorizing it correctly (visualize or say)}
+	\scoreitem[4]{30}{Picking up breakfast items for transportation to the table}
+	\scoreitem[4]{30}{Placing breakfast items on the table}
+	\scoreitem{100}{Pouring cereal into the bowl}
```

Comment:
why do we reduce the amount of points for pouring? This is still very challenging for many teams. I think 300 pts is appropriate for such a challenging task.

Reply:
I agree. I would leave it at 300 and maybe increase the penalty for spilling? Because pouring without spilling is still very challenging but just pouring can be potentially done closed loop. 
Also maybe add the entire package must be poured or add penalty for not pouring entire contents. 
Same comments go for the milk pouring in bonus section.